stdlib/database/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -96,6 +96,21 @@ public type Client client object {
         }
     }
 
+    # Executes a SQL stored procedure and returns the result as stream and execution summary.
+    #
+    # + sqlQuery - The query to executed the SQL stored procedure.
+    # + rowTypes - The array of `typedesc` of the records that should be returned as a result. If this is not provided
+    #               the default column names of the query result set be used for the record attributes.
+    # + return - Summary of the execution is returned in `ProcedureCallResult` or `sql:Error`.
+    public remote function call(@untainted sql:ParameterizedCallQuery sqlQuery, typedesc<record {}>[] rowTypes = [])
```

Comment:
The convention of using full stop is not properly followed in the api doc statements. Refer below sample by @MaryamZi . No full stop at the end of first sentence of the parameter and return type description and it is there only at function description. Maybe u can fix the newly added docs in this PR and sent another one to fix all the places in db modules in a separate PR as there is a lot of places to fix.  

```ballerina
# Description for the function.
#
# + i - One sentence only
# + s - Sentence one. Sentence two.
# + return - Return description
public function foo(int i, string s) returns boolean {
    return true;
}
```

Reply:
Fixed in c8978fd